cf/data/array/mixin/compressedarraymixin.py

```diff
@@ -49,3 +49,84 @@ def _lock_file_read(self, array):
             array = da.from_array(array, chunks=chunks, lock=True)
 
         return array
+
+    def to_dask_array(self, chunks="auto"):
+        """Convert the data to a `dask` array.
+
+        .. versionadded:: 3.16.0
```

Comment:
```suggestion
        .. versionadded:: 3.14.0
```

Since it technically appeared in `3.14.0` (as I see from the removal of the equivalent in, e.g., `cf/data/array/gatheredarray.py`), only it has moved to a new location under-the-hood?